src/graphql/resolvers/mutations/milestones.js

```diff
@@ -1,11 +1,16 @@
+import milestoneStore from '../../../dataSources/cloudFirestore/milestone';
+
 export const fieldResolvers = {
   MilestonesMutation: {
-    create: ({ eventId }, { milestone }, { dataSources: { firestore } }) => {
-      throw new Error('Not implemented yet.');
-    },
-    delete: ({ eventId }, { id }, { dataSources: { firestore } }) => {
-      throw new Error('Not implemented yet.');
-    },
+    create: async (
+      { eventId },
+      { milestone },
+      { dataSources: { firestore } },
+    ) => milestoneStore(firestore).create(eventId, milestone),
+
+    delete: async ({ eventId }, { id }, { dataSources: { firestore } }) =>
```

Comment:
this doesn't need to be async. Just return the function. Also, do we have the right security assigned in the schema for delete? I ask because we didn't have this implemented before.

Reply:
@csell5 Schema :
```graphql
type MilestonesMutation {
  create(milestone: MilestoneCreateInput!): Milestone! @auth(requires: "events")
  delete(id: ID!): ID @auth(requires: "events")
  milestone(id: ID!): MilestoneMutation @auth(requires: "events")
}
```

I am assuming the create doesn't require an async as well then.

Reply:
I assume we're only getting events then. but 👍

Reply:
I am not following what you mean by that comment